cypress/support/step_definitions/dictionary/copyDictionary/copyDictionary.ts

```diff
@@ -0,0 +1,31 @@
+import 'cypress-wait-until';
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser } from "../../../utils";
+
+Given("the user is on the dictionary page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/`);
+  cy.findByText("Versions").should("be.visible");
+});
+When("the user clicks the more actions button", () =>
+  cy.findByTitle("More actions").click()
+);
+When(/the user selects the "(.+)" menu list item/, menuItem =>
+  cy.findByText(menuItem).click()
+);
+When("the user is on copy dictionary form", () =>
+  cy.url().should("contain",`/collections/new/`)
```

Comment:
Can we add the `copyFrom` parameter so that we confirm that it is actually copying? For example when you are copying the URL is slightly different from normal creating `https://openmrs.openconceptlab.org/collections/new/?copyFrom=/users/haddy315/collections/hd-1/` .

Reply:
I actually advised Suruchi to do this, largely because including the `copyFrom` parameter seemed to cause the assertion to fail for no good reason. It's definitely not ideal, but it works...